PIPs/pip-21.md

```diff
@@ -0,0 +1,21 @@
+---
+pip: 21
+title: unbond token can rebond at any time. 
+author: hui (@laosiji-io)
+status: Draft
+type: Standards
+category: validator
+created: 15-01-2024
+---
+
+## Abstract
+
+The purpose of this proposal is to make the staking token more liquid.
+
+## Motivation
+
+Because the time of unbond_interval takes too long, and when the token is unboned, these token will not have any effect.
```

Comment:
Please add more details for motivation and specification

Reply:
added what I have temporarily thought of